test-configs.yaml

```diff
@@ -1040,11 +1040,32 @@ device_types:
     class: arm64-dtb
     boot_method: uboot
 
+# sun50i-h6-pine-h64 model A
   sun50i-h6-pine-h64:
     mach: allwinner
     class: arm64-dtb
     boot_method: uboot
 
+  sun50i-h6-pine-h64-model-b:
+    mach: allwinner
+    class: arm64-dtb
+    boot_method: uboot
+
+#Pine H64 exists in two version A and B.
+#Before 5.5, sun50i-h6-pine-h64 was for model B but could be also used for model A.
+#Starting with 5.5, sun50i-h6-pine-h64 will be only for model A and a new
+#dtb will exists for model B.
+#And so a new LAVA device-type will be necessary.
+#But we need to test model B before 5.5 using the pre 5.5 sun50i-h6-pine-h64.dtb WITH the the new model B LAVA device-type.
```

Comment:
Maybe indent the comments to align with the device definition and add a space after the `#` to make it easier to read.